var/spack/repos/builtin/packages/ncview/package.py

```diff
@@ -25,7 +25,13 @@ def configure_args(self):
 
         config_args = []
 
+        # Problems on some systems (e.g. NASA Discover with Intel)
+        # to find udunits include and library files despite
+        # dependency being specified above
+        config_args.append("--with-udunits2_incdir={}".format(spec["udunits"].prefix.include))
+        config_args.append("--with-udunits2_libdir={}".format(spec["udunits"].prefix.lib))
```

Comment:
Is the current issue with udunits not being found a deficiency in the udunits package/module, or an issue with how ncview is finding it?

Reply:
I don't know, ncview should find udunits just fine because the `depends_on(udunits)` is in the spack package, but for some reason on discover with Intel (and only there) it doesn't find it. Adding those two lines doesn't hurt. I verified that on the other systems, even without those two lines, `ncview` is finding the correct (spack) `udunits`, not some other installations in a default path.

Reply:
Do you know whether other packages that depend on it have had the same issue?

Reply:
No, I don't, and we don't have any other package in the stack that use udunits.